app/cdap/components/GridTable/components/GridHeaderCell/index.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Box, Card, styled, TableCell, Typography } from '@material-ui/core';
+import { TypographyComponent } from '../Typography';
+import { useGridHeaderCellStyles } from './styles';
+import { IGridHeaderCellProps } from './types';
+
+const StringIndicatorBox = styled(Box)({
+  display: 'flex',
+});
+
+export const GridHeaderCell: React.FC<IGridHeaderCellProps> = ({ label, types }) => {
```

Comment:
Let typescript infer this file type -- don't use React.FC

Reply:
Done